file/file.go

```diff
@@ -23,9 +23,9 @@ func (f *Info) Sort(file io.Reader) error {
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
 		text := scanner.Text()
-		err := vector.Sort(ans, text)
+		err := vector.Sort(ans, text, "\t", 0)
```

Comment:
I don't think we want the separator hard coded. Maybe we could pass it as a field in *Info